### ISSUE TITLE
TextWire.getEscapingQuotes() generates garbage

### ISSUE THREAD

**Issue by agusevas** [OpenHFT/Chronicle-Wire#377]
In my application I use Chronicle Wire and parse JSON messages containing many string-escaped numbers (i.e. `"0.1234"`). It is the same use case as shown in example under https://chronicle.software/how-to-build-low-latency-crypto-trading-system-using-java-and-chronicle-services/. 

```
    // Binance provides prices / qtys in escape quotes, so we have to read a text first
    private double readDoubleInQuotes(ValueIn valueIn) {
        doubleText.setLength(0);
        valueIn.text(doubleText);
        // StringUtils from Chronicle-Services allows to avoid allocating a String
        return StringUtils.parseDouble(doubleText);
    }
```

Profiling my application I have identified, that the `TextWire.getEscapingQuotes()` method allocates new memory for `StopCharTesters.QUOTES::escaping` supplier. I have tested with Java 11 and Java 17 with similar results. Here is the snapshot from the Async Profiler using Java 17:

<img width="1830" alt="CleanShot 2022-01-31 at 09 03 36@2x" src="https://user-images.githubusercontent.com/3429466/151758376-318d3b4a-2154-4bee-bf38-93d4095df402.png">

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.23ea3](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.23ea3), [BOM-2.23ea16](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea16)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.23ea3](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.23ea3), [BOM-2.23ea17](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea17)
